lib/request.js

```diff
@@ -40,7 +40,10 @@ const request = (options, success, error) => {
 
       res.on('data', (chunk) => data += chunk);
       res.on('end', () => {
-        const parsed = JSON.parse(data);
+        let parsed;
+        if (data !== '') {
+          parsed = JSON.parse(data);
```

Comment:
Let's replace this if-block it with ternary-operator:
```js
const parsed = data !== '' ? JSON.parse(data) : '';
```

Reply:
I prefer to pass `null` in case of empty body